packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt

```diff
@@ -0,0 +1,46 @@
+Enable single sign-on for tab applications
+-------------------------
+Microsoft Teams provides a mechanism by which an application can obtain the signed-in Teams user token to access Microsoft Graph (and other APIs).
+Teams Toolkit facilitates this interaction by abstracting some of the Azure Active Directory flows and integrations behind some simple, high level APIs.
+This enables you to add single sign-on (SSO) features easily to your Teams application.
+
+Changes to your project
+-------------------------
+When you added the SSO feature to your application, Teams Toolkit updated your project to support SSO:
+After you successfully added SSO into your project, Teams Toolkit will create and modify some files that helps you implement SSO feature.
+1. Create: 'aad.template.json' under 'templates/appPackage'
+   - The Azure Active Directory application manifest that is used to register the application with AAD.
+2. Modify: 'manifest.template.json' under 'templates/appPackage'
+   - An 'webApplicationInfo' object will be added into your Teams app manifest template. This field is required by Teams when enabling SSO.
+3. Create: 'Auth/tab'
+   - Reference code and a 'README.txt' file. These files are provided for reference. See below for more information.
+
```

Comment:
Will we update appsettings.json for user?

Reply:
Added logic and readme in the new commit, thx!